tests/snapshots/tests.contrib.sanic.test_sanic_server.test_sanic_errors_pre_21.9.json

```diff
@@ -54,6 +55,7 @@
       "span.kind": "server"
     },
     "metrics": {
+      "_dd.measured": 1,
```

Comment:
Marking spans are measured will ensure trace metrics are computed. This is something we can call out in a release note. With this change user's might see a change in their trace metrics.

Reply:
AFAIK since these spans are top-level, they already have metrics computed. @wantsui had also noted this in a resolved thread. But now we are explicitly saying they are measured.